The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/9132140147/job/25112835515) [(plain)](https://github.com/rust-lang/rust/commit/f0b8da4430971ce1a34848852849b8b4e3f20778/checks/25112835515/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
Getting action download info
Download action repository 'msys2/setup-msys2@v2.22.0' (SHA:cc11e9188b693c2b100158c3322424c4cc1dadea)
Download action repository 'actions/checkout@v4' (SHA:0ad4b8fadaa221de15dcec353f45205ec38ea70b)
Download action repository 'actions/upload-artifact@v4' (SHA:65462800fd760344b1a7b4382951275a0abb4808)
Complete job name: PR - mingw-check-tidy
git config --global core.autocrlf false
shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
---
COPY scripts/sccache.sh /scripts/
RUN sh /scripts/sccache.sh

COPY host-x86_64/mingw-check/reuse-requirements.txt /tmp/
RUN pip3 install --no-deps --no-cache-dir --require-hashes -r /tmp/reuse-requirements.txt \
    && pip3 install virtualenv
COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/

# NOTE: intentionally uses python2 for x.py so we can test it still works.
# NOTE: intentionally uses python2 for x.py so we can test it still works.
# validate-toolstate only runs in our CI, so it's ok for it to only support python3.
ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test \
           --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
# This file is autogenerated by pip-compile with Python 3.10
# by the following command:
#
#    pip-compile --allow-unsafe --generate-hashes reuse-requirements.in
---

#12 [5/8] COPY host-x86_64/mingw-check/reuse-requirements.txt /tmp/
#12 DONE 0.0s

#13 [6/8] RUN pip3 install --no-deps --no-cache-dir --require-hashes -r /tmp/reuse-requirements.txt     && pip3 install virtualenv
#13 0.427   Downloading binaryornot-0.4.4-py2.py3-none-any.whl (9.0 kB)
#13 0.444 Collecting boolean-py==4.0
#13 0.450   Downloading boolean.py-4.0-py3-none-any.whl (25 kB)
#13 0.466 Collecting chardet==5.1.0
---
#13 3.691 Building wheels for collected packages: reuse
#13 3.692   Building wheel for reuse (pyproject.toml): started
#13 4.012   Building wheel for reuse (pyproject.toml): finished with status 'done'
#13 4.014   Created wheel for reuse: filename=reuse-1.1.0-cp310-cp310-manylinux_2_35_x86_64.whl size=181117 sha256=f5f58750481f69515c2c0d1d503daf565e2565c370d07fc6aeb95fe3498b4269
#13 4.014   Stored in directory: /tmp/pip-ephem-wheel-cache-0nt64cjh/wheels/c2/3c/b9/1120c2ab4bd82694f7e6f0537dc5b9a085c13e2c69a8d0c76d
#13 4.016 Installing collected packages: boolean-py, binaryornot, setuptools, reuse, python-debian, markupsafe, license-expression, jinja2, chardet
#13 4.038   Attempting uninstall: setuptools
#13 4.038     Found existing installation: setuptools 59.6.0
#13 4.039     Not uninstalling setuptools at /usr/lib/python3/dist-packages, outside environment /usr
#13 4.039     Not uninstalling setuptools at /usr/lib/python3/dist-packages, outside environment /usr
#13 4.040     Can't uninstall 'setuptools'. No files were found to uninstall.
#13 4.700 Successfully installed binaryornot-0.4.4 boolean-py-4.0 chardet-5.1.0 jinja2-3.1.2 license-expression-30.0.0 markupsafe-2.1.1 python-debian-0.1.49 reuse-1.1.0 setuptools-66.0.0
#13 4.700 WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
#13 5.220 Collecting virtualenv
#13 5.285   Downloading virtualenv-20.26.2-py3-none-any.whl (3.9 MB)
#13 5.444      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.9/3.9 MB 25.0 MB/s eta 0:00:00
#13 5.484 Collecting distlib<1,>=0.3.7
#13 5.490   Downloading distlib-0.3.8-py2.py3-none-any.whl (468 kB)
#13 5.501      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 KB 53.9 MB/s eta 0:00:00
#13 5.534 Collecting platformdirs<5,>=3.9.1
#13 5.541   Downloading platformdirs-4.2.2-py3-none-any.whl (18 kB)
#13 5.575 Collecting filelock<4,>=3.12.2
#13 5.581   Downloading filelock-3.14.0-py3-none-any.whl (12 kB)
#13 5.667 Installing collected packages: distlib, platformdirs, filelock, virtualenv
#13 5.832 Successfully installed distlib-0.3.8 filelock-3.14.0 platformdirs-4.2.2 virtualenv-20.26.2
#13 DONE 5.9s

#14 [7/8] COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
#14 DONE 0.0s
---
DirectMap4k:      198592 kB
DirectMap2M:     6092800 kB
DirectMap1G:    12582912 kB
##[endgroup]
Executing TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
+ TIDY_PRINT_DIFF=1 python2.7 ../x.py test --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
    Finished `dev` profile [unoptimized] target(s) in 0.03s
##[endgroup]
downloading https://ci-artifacts.rust-lang.org/rustc-builds-alt/550d1b4fb6de23990f4108815c3b1a9d1659e5c4/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz
extracting /checkout/obj/build/cache/llvm-550d1b4fb6de23990f4108815c3b1a9d1659e5c4-true/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz to /checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm
---
   Compiling tidy v0.1.0 (/checkout/src/tools/tidy)
    Finished `release` profile [optimized] target(s) in 25.53s
##[endgroup]
fmt check
##[error]Diff in /checkout/tests/run-make/no-intermediate-extras/rmake.rs at line 8:
 
 fn main() {
 fn main() {
-    rustc().crate_type("rlib").arg("--test").input("foo.rs").run(); 
+    rustc().crate_type("rlib").arg("--test").input("foo.rs").run();
     match fs::remove_file(tmp_dir().join("foo.bc")) {
         Ok(_) => {
             println!("An unwanted .bc file was created by run-make/no-intermediate-extras.");
##[error]Diff in /checkout/tests/run-make/no-intermediate-extras/rmake.rs at line 15:
             std::process::exit(0);
+        }
         Err(e) => {
             std::process::exit(1);
         }
         }
Running `"/checkout/obj/build/x86_64-unknown-linux-gnu/rustfmt/bin/rustfmt" "--config-path" "/checkout" "--edition" "2021" "--unstable-features" "--skip-children" "--check" "/checkout/tests/run-make/rustdoc-scrape-examples-remap/rmake.rs" "/checkout/tests/run-make/alloc-no-oom-handling/rmake.rs" "/checkout/tests/run-make/non-unicode-in-incremental-dir/rmake.rs" "/checkout/tests/run-make/exit-code/rmake.rs" "/checkout/tests/run-make/print-cfg/rmake.rs" "/checkout/tests/run-make/wasm-stringify-ints-small/rmake.rs" "/checkout/tests/run-make/alloc-no-sync/rmake.rs" "/checkout/tests/run-make/arguments-non-c-like-enum/rmake.rs" "/checkout/tests/run-make/no-intermediate-extras/rmake.rs" "/checkout/tests/run-make/rustdoc-target-spec-json-path/rmake.rs" "/checkout/tests/run-make/rust-lld-custom-target/rmake.rs" "/checkout/tests/run-make/wasm-symbols-not-exported/rmake.rs" "/checkout/tests/run-make/wasm-abi/rmake.rs" "/checkout/tests/run-make/doctests-keep-binaries/rmake.rs"` failed.
If you're running `tidy`, try again with `--bless`. Or, if you just want to format code, run `./x.py fmt` instead.
  local time: Fri May 17 17:44:39 UTC 2024
  network time: Fri, 17 May 2024 17:44:39 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/tests/run-make/no-intermediate-extras/rmake.rs
+++ b/tests/run-make/no-intermediate-extras/rmake.rs
@@ -13,10 +13,10 @@ fn main() {
     match fs::remove_file(tmp_dir().join("foo.bc")) {
         Ok(_) => {
             println!("An unwanted .bc file was created by run-make/no-intermediate-extras.");
-            std::process::exit(0);
+            std::process::exit(1);
         }
         Err(e) => {
-            std::process::exit(1);
+            std::process::exit(0);
         }
     }
 }

--- a/tests/run-make/no-intermediate-extras/rmake.rs
+++ b/tests/run-make/no-intermediate-extras/rmake.rs
@@ -6,6 +6,7 @@
 //@ ignore-cross-compile
 
 use run_make_support::{rustc, tmp_dir};
+use std::fs;
 
 fn main() {
     rustc().crate_type("rlib").arg("--test").input("foo.rs").run();

--- a/tests/run-make/no-intermediate-extras/rmake.rs
+++ b/tests/run-make/no-intermediate-extras/rmake.rs
@@ -8,12 +8,12 @@
 use run_make_support::{rustc, tmp_dir};
 
 fn main() {
-    rustc().crate_type("rlib").arg("--test").input("foo.rs").run(); 
+    rustc().crate_type("rlib").arg("--test").input("foo.rs").run();
     match fs::remove_file(tmp_dir().join("foo.bc")) {
         Ok(_) => {
             println!("An unwanted .bc file was created by run-make/no-intermediate-extras.");
             std::process::exit(0);
-        },
+        }
         Err(e) => {
             std::process::exit(1);
         }

